terracommon/datastore/fields.py

```diff
@@ -38,13 +42,11 @@ def to_internal_value(self, data):
 
         else:
             try:
-
                 decoded = base64.b64decode(encoded)
```

Comment:
This seems to be robed from https://github.com/Terralego/terra-back/pull/18/files ?

Reply:
Yes, or the inverse :)